### PR TITLE
Reco add new type of alerts

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 1]
```diff
@@ -474,12 +581,45 @@ def parse_table_row_to_dict(alert: List[Dict[str, Any]]) -> Dict[str, Any]:
             if parsed_time:
                 obj[key] = parsed_time.strftime(DEMISTO_OCCURRED_FORMAT)
         if key == "risk_level":
-            obj[key] = int(obj[key])
+            try:
+                obj[key] = int(obj[key])
+            except Exception:
+                obj[key] = str(obj[key])
```
Comment (samuelFain) [inline Packs/Reco/Integrations/Reco/Reco.py:587]: Why is the expression in the exception handling (under `except`) is the same as the expression under `try`?

**Review by samuelFain — COMMENTED** [review]
Hi @GalNakash-RecoLabs, and thank you for your contribution!
Please see me comments.
Also, there are a few validation issues that need addressing:
- The docker image tag is not the latest numeric tag, please update it.
The docker image tag in the yml file is: `3.10.10.51930`
The latest docker image tag in docker hub is: `3.10.11.57293`
You can update the docker image manually or via `demisto-sdk` by running the following command:
`demisto-sdk format -ud -i Packs/Reco/Integrations/Reco/Reco.yml`
- It seems the `reco-get-sensitive-assets-by-id` command is missing, which can break backwards compatibility.
It's kind of weird because this command do exist in the current version in content, but I dont see the removing of this command in the diffs or comments in this PR.
Did you mean to remove this command?
- The pack version (currently: 1.0.4) needs to be raised - make sure you are merged from master and update the "currentVersion" field in the pack_metadata.json and make sure to [update the pack's release notes](https://xsoar.pan.dev/docs/documentation/release-notes#format) with the changes made in this PR.
You can run the following command: `demisto-sdk update-release-notes -i Packs/Reco -u minor`.

In the meanwhile, feel free to contact me for any questions/clarifications.

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 2]
```diff
@@ -43,12 +44,12 @@ def __init__(self, api_token: str, base_url: str, verify: bool, proxy):
...
-        source: Optional[str] = None,
-        before: Optional[datetime] = None,
-        after: Optional[datetime] = None,
-        limit: int = 1000,
+            self,
+            risk_level: Optional[int] = None,
+            source: Optional[str] = None,
+            before: Optional[datetime] = None,
+            after: Optional[datetime] = None,
+            limit: int = 1000,
```
[the remark this answers is not in the document]
Comment (GalNakash-RecoLabs) [reply Packs/Reco/Integrations/Reco/Reco.py]: reco-get-sensitive-assets-by-id - THAT'S ODD, let me check it

**Comment by GalNakash-RecoLabs** [comment]
Fixed all comments

**Comment by samuelFain** [comment]
@GalNakash-RecoLabs Nice work!
A few comments:

- Our secrets detection workflow has detected a possible sensitive value, If this value is not sensitive and is needed in the code or yml file, please add it to the pack's `.secrets-ignore` file, otherwise its preferred you change or remove it:
>Running secrets validation against origin/master
Secrets were found in the following files:
In File: Packs/Reco/Integrations/Reco/Reco_test.py:106
Secret found: gal@reco.ai
In File: Packs/Reco/Integrations/Reco/Reco_test.py:113
Secret found: gal@reco.ai

- There are a few flake8 errors which are failiing the linting checks, please address them:
>Reco_test.py:383:81: W504 line break after binary operator
Reco_test.py:412:81: W504 line break after binary operator
Reco_test.py:441:81: W504 line break after binary operator

- About the demo - let me know whether you want to record and share with me a demo video, or if you want to schedule a meeting for a live demo, whichever works best for you.

Thanks!

**Comment by GalNakash-RecoLabs** [comment]
Done, when would be a good time to schedule demo?

**Comment by GalNakash-RecoLabs** [comment]
@samuelFain @edik24 should we schedule 15 mins for the  demo?

**Review by samuelFain — APPROVED** [review]

**Pull request merged**